README.md

```diff
@@ -0,0 +1,22 @@
+server: the DB + "manager"
+==========================
+
+Communicates with the controllers.
+
+Setup
+-----
+
+**Required Python version: 3.4**
+
+1. Create virtualenv if necessary: `pyvenv ./venv` (or `pyvenv-3.4 ./venv`)
+2. Activate virtualenv:
+   - bash, zsh: `source venv/bin/activate`
+   - fish: `. venv/bin/activate.fish`
+   - csh, tcsh: `source venv/bin/activate.csh`
+3. Install dependencies if necessary: `pip install -r requirements.txt`
+4. Configure by setting environment variables: `PORT` and `DB_URL`.  
+   You can also write it to `.env`, `./manage.py run` will load it.
+   `cp .env{.example,}; $EDITOR .env` to get started.
```

Comment:
Hmm, co ja viem ci `.env` je dobry nazov. Radsej by som nieco co nezacina na bodku. Typu `env` alebo `env.ini` alebo `settings.ini` alebo `settings.py` (ak casom bude zlozitejsi) alebo co ja viem.

Hmm, co ja viem ci konfiguracia aj cez envvars. Imho staci subor, a je tak jasnejsi single source of truth.